.github/workflows/golangci-lint.yaml

```diff
@@ -21,8 +21,8 @@ jobs:
         # working-directory: somedir
 
         # Optional: golangci-lint command line arguments.
-        # TODO: remove --skip-files controllers/suite_test.go once logic is implemented
-        args: -E gosec --timeout=6m --skip-files controllers/suite_test.go
+        # TODO: remove --skip-files controllers/suite_test.go api/v1alpha1/webhook_suite_test.go once logic is implemented
+        args: -E gosec --timeout=6m --skip-files controllers/suite_test.go,api/v1alpha1/webhook_suite_test.go
```

Comment:
Why are we skipping tests instead of implementing it? At the every least we need to mark it as TODO and create a tracker issue.

Reply:
It already a TODO to remove these skip-files once we implement tests. We can create a issue.